.github/workflows/release.yml

```diff
@@ -19,22 +19,22 @@ jobs:
 
   release:
     runs-on: ubuntu-20.04
-    container:
-      image: xaynetci/yellow:9135de0
+    #container:
```

Comment:
docker image does not have rsync that is needed by `release.sh`

Reply:
I think since we had the problem with the size no one run the release CI and we didn't notice.

@rustonaut we have setup-job that install everything that is still used on the mac runner, and we have setup-job-docker that only loads env and setup cache for when we run on docker.